src/Tiles/Providers/RosreestrProvider.h

```diff
@@ -32,7 +32,7 @@ class RosreestrProvider : public BaseProvider
 	{
 		Id = tkTileProvider::Rosreestr;
 		Name = "Rosreestr";
-		_copyright = "����������";
+		_copyright = "�?in?aano?";

```

Comment:
I'm not sure what to make of this copyright, but I don't think we should be changing it. Do you know something I don't know here?

Reply:
Oh, I didn't realize that. I think it changed me about the coding. I don't know this service rosreestr, so I didn't want to change anything here.

Reply:
Oh, I didn't realize that. I think it changed me about the coding. I don't know this service rosreestr, so I didn't want to change anything here.